lomap/graphgen.py

```diff
@@ -901,8 +893,8 @@ def layout_info(self):
             morph_data = "morph_pairs = "
         with open(self.dbase.options['name'] + "_score_with_connection.txt", "w") as info_txt:
             all_key_id = self.dbase.dic_mapping.keys()
-            data = ["%-10s,%-10s,%-25s,%-25s,%-15s,%-15s,%-15s,%-10s\n" % (
-            "Index_1", "Index_2", "Filename_1", "Filename_2", "Str_sim", "Eff_sim", "Loose_sim", "Connect")]
+            data = ["%-10s,%-10s,%-25s,%-25s,%-15s,%-15s,%-15s,%-10s,%s\n" % (
+            "Index_1", "Index_2", "Filename_1", "Filename_2", "Str_sim", "Eff_sim", "Loose_sim", "Connect", "Map_str")]
```

Comment:
this change looks like it's going to change the output format, which might break other user's usage

Reply:
Sorry for my being paranoid.
How about declare the broken change in output file format in the release note of next version?
If you insist this small fix is OK to tolerate, I will roll back it quickly.

Reply:
We can't break the output format, even if it does improve things.  This change will have to be removed.

Reply:
Change reverted.